Add new section/procedural steps to enable Capsule in the web UI afte… (#373)
theforeman/foreman-documentation: guides/common/modules/proc_enabling-capsule-in-UI.adoc

[id="assigning-organization-location-capsule-server_{context}"]

> = Enabling the {SmartProxyServer} in the {Project} web UI

Reviewer: What do you mean by Enabling? 
In this procedure, we assign {SmartProxy} to a correct organization and location. Please change the title.

Author: Changed

After installing the {SmartProxyServer} packages, you must enable the {SmartProxy} in the {Project} web UI by assigning the correct organization and location to it.

.Procedure

. Log into the {Project} web UI.

> . Click *Organization*, which is in the upper-left of the UI, then select *Any Organization*.

Author: Done

> . Click *Location*, which is in the upper-left of the UI, then select *Any Location*.

Author: Done

> . Click *Hosts* > *All Hosts* and select the checkbox next to the {SmartProxyServer}. 

Author: Fixed

> . Click the *Select Actions* list and select *Assign Organization*. 

Author: Done

> . Select organization from the *Organization* list, click *Fix Organization on Mismatch*, then click *Submit*. 

Author: Done

. Select the checkbox next to the {SmartProxyServer}, click the *Select Actions* list, and select *Assign Location*. 

> . Select location from the *Location* list and click *Fix Location on Mismatch*, then click *Submit*.

Author: Done

> . Click *Administer* > *Organizations* and click on the organization name where you want to see the {SmartProxyServer}.

Reviewer: ```suggestion
. Navigate to *Administer* > *Organizations* and click the organization to which you have assigned {SmartProxy}.
```
From the IBM Style Guide:
Use “click,” not “click on,” to refer to the action of selecting something on a graphical interface.

Author: Fixed

> . Click *{SmartProxies}* on the left-side menu and ensure the {SmartProxyServer} is listed under the *Selected items* list, then click *Submit*.

Reviewer: ```suggestion
. Click the *{SmartProxies}* tab and ensure that the {SmartProxy} is listed under the *Selected items* list, then click *Submit*.
```

> . Click *Administer* > *Locations* and click on the location name where you want to see the {SmartProxyServer}.

Reviewer: Same as above.

Author: Done

> . Click *{SmartProxies}* on the left-side menu and ensure the {SmartProxyServer} is listed under the *Selected items* list, then click *Submit*.

Reviewer: Same as above.

> You have now enabled the {SmartProxyServer} in the {Project} web UI. Select the correct organization and location in the {Project} web UI, and verify if the {SmartProxyServer} is listed in the following locations by navigating to:

Reviewer: What does enable mean here?

Reviewer 2: We do not use these half sentences with a colon either. We use a full sentence to introduce a list. 
The first sentence is redundant and can be safely removed.

Author: Changed

. From the *Select Actions* list, select *Assign Organization*. 
. From the *Organization* list, select the organization where you want to assign this {SmartProxy}. 
. Click *Fix Organization on Mismatch*. 
. Click *Submit*. 
. Select the {SmartProxyServer}. From the *Select Actions* list, select *Assign Location*.
. From the *Location* list, select the location where you want to assign this {SmartProxy}.
. Click *Fix Location on Mismatch*.
. Click *Submit*.
. Navigate to *Administer* > *Organizations* and click the organization to which you have assigned {SmartProxy}.
. Click the *{SmartProxies}* tab and ensure that the {SmartProxyServer} is listed under the *Selected items* list, then click *Submit*.
. Navigate to *Administer* > *Locations* and click the location to which you have assigned {SmartProxy}.
. Click the *{SmartProxies}* tab and ensure that the {SmartProxyServer} is listed under the *Selected items* list, then click *Submit*.

Select the organization and location in the {Project} web UI, and verify if the {SmartProxyServer} is listed in the following locations.

* Navigate to *Hosts* > *All Hosts*.

> * Navigate to *Infrastructure* > *{SmartProxies}*.

Reviewer 2: Sorry for all the nitpicking.
`Select the organization and location in the {Project} web UI, and verify if the {SmartProxyServer} is listed in the following locations.` - this is a procedural step, not a paragraph. 
 Bullets are not used for sequential steps.

I understand that the procedure is complete. Perhaps this might be better placed in a Conclusion section where they can optionally verify as this is not part of the procedure, and I can understand the difficulty you're having expressing this.

Author: Thanks, done.

> You have now assigned the correct organization and location to {SmartProxyServer}. Select the organization and location in the {Project} web UI, and verify if the {SmartProxyServer} is listed in the following locations by navigating to:

Reviewer 2: As I said, this `
You have now assigned the correct organization and location to {SmartProxyServer}.` is redundant. We also don't have these open half-sentences with the colon. We always wrote a full sentence. Nowhere in mod docs or in IBM or in the existing Sat docs have I seen this style.

Author: Didn't see your comment earlier. Fixed now.

Author: @melcorr Didn't see your comment earlier; fixed now. Thanks.

[id="assigning-organization-location-capsule-server_{context}"]

= Assigning the Correct Organization and Location to {SmartProxyServer} in the {Project} web UI

> After installing the {SmartProxyServer} packages, you must enable the {SmartProxy} in the {Project} web UI by assigning the correct organization and location to it.

Reviewer 3: Is this still true? I thought https://projects.theforeman.org/issues/26092  changed it that if there's only one org (or location) that's picked automatically. The Smart Proxy is created by the installer via the API. That means `must` is only correct if there's more than one organization/location.

If my theory is correct, it should do the right thing since Foreman 1.23. IIRC that's Satellite 6.7 but I'd recommend testing on the latest version.

Author: Thanks @ekohl. I've added text for clarity.

> After installing the {SmartProxyServer} packages, if there is more than one organization or location, you must assign the organization and location to {SmartProxy} to correctly enable {SmartProxy} in the {Project} web UI.

Reviewer: we do not enable anything here, we make the Capsule visible. The word enable is missleading.

Author: Fixed

After installing the {SmartProxyServer} packages, you must enable the {SmartProxyServer} in the {Project} web UI by assigning the correct organization and location to it.

.Procedure

. Log into the {Project} web UI.
. Click *Organization*, then select *Any Organization*.
. Click *Location*, then select *Any Location*.
. Click *Hosts* > *All Hosts* and select the checkbox next to the {SmartProxyServer}. 
. Click the *Select Actions* list and select *Assign Organization*. 
. Select organization from the *Organization* list, click *Fix Organization on Mismatch*, then click *Submit*. 
. Select the checkbox next to the {SmartProxyServer}, click the *Select Actions* list, and select *Assign Location*. 
. Select location from the *Location* list and select *Fix Location on Mismatch*, then click *Submit*.
. Click *Administer* > *Organizations* and click on the organization name where you want to see the {SmartProxyServer}.
. Click *Capsules* on the left-side menu and ensure the {SmartProxyServer} is listed under the *Selected items* list, then click *Submit*.
. Click *Administer* > *Locations* and click on the location name where you want to see the {SmartProxyServer}.

> . Click *Capsules* on the left-side menu and ensure the {SmartProxyServer} is listed under the *Selected items* list, then click *Submit*.

Reviewer 2: Please use variables.

Author: Fixed

. Select the correct organization and location and verify if the {SmartProxyServer} is listed under the following locations by navigating to:
* Click *Hosts* > *All Hosts*.

> * Click *Infrastructure* > *Capsules*.

Reviewer 2: Please use variables.

Author: Fixed

> ////

Reviewer 2: Why the commented out bits here?

Author: Oops, this was meant to be a WIP PR/commit; though didn't mean to push the commented out bits. Fixed, thanks!
